packages/components/psammead-embed-error/src/__snapshots__/index.test.jsx.snap

```diff
@@ -72,9 +72,11 @@ exports[`EmbedError renders a default embed error 1`] = `
   <div
     class="c1"
   >
-    <strong>
-      Sorry, we can't display this part of the story on this lightweight mobile page.
-    </strong>
+    <div>
+      <strong>
+        Sorry, we can't display this part of the story on this lightweight mobile page.
+      </strong>
+    </div>
```

Comment:
`div` is a block element, which means we can remove `display: block;` from the `strong` element. 👍